Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Menu/Menu.bind

```diff
@@ -0,0 +1,167 @@
+<Page x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.MenuPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:commands="using:Microsoft.Toolkit.Uwp.SampleApp.Menu.Commands"
+      xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      mc:Ignorable="d">
+
+    <Page.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ResourceDictionary Source="MenuStyles.xaml" />
+            </ResourceDictionary.MergedDictionaries>
+
+            <commands:NewProjectCommand x:Key="NewProject" />
+            <commands:NewFileCommand x:Key="NewFile" />
+            <commands:GenericCommand x:Key="GenericCommand" />
+        </ResourceDictionary>
+
+
+    </Page.Resources>
+
+    <Grid Background="{StaticResource Brush-Grey-05}">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="auto" />
+            <RowDefinition Height="auto" />
+        </Grid.RowDefinitions>
+
+        <controls:Menu HeaderButtonStyle="{StaticResource MenuHeaderButtonStyle}"
+                       MenuFlyoutStyle="{StaticResource MenuStyle}"
```

Comment:
Great question @mdtauk. I tested this and the menu flyout does respect the requested theme but the menu itself does not.